activerecord-id_regions.gemspec

```diff
@@ -20,8 +20,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activerecord", ">= 5.0", "< 5.3"
```

Comment:
Do we have to drop support for Rails 5?  I'm thinking this can be loosened to `">= 5.0", "< 7.0"`

Reply:
@bdunne yeah, I guess that is a good point, since we aren't changing anything else.  I will make that change. 👍